changelog.d/issue-8817

```diff
@@ -0,0 +1,12 @@
+synopsis: Neutral field to add files to sdist
+packages: Cabal-syntax
+prs: #10107
+issues: #8817
+significance: significant
+
+description: {
+  Adds the `extra-files` field to the cabal file specification. This is like
+  the other `extra-*` fields (e.g. `extra-source-files`, `extra-doc-files`)
+  exception it does _not_ add any other semantics (cf. `extra-source-files`

```

Comment:
```suggestion
  except it does _not_ add any other semantics (cf. `extra-source-files`
```

Also, it's be good to seize the opportunity to mention how extra-doc-files are special (they get copied by Haddock to the html directory).

Reply:
Do you want it in the issue? Or somewhere else (too)?

Reply:
Only in this file. It will get to the top of release notes (due to `significance`), so, pretty visible…

Reply:
Got it, done.

Reply:
Perfect, thank you!